packages/fetchai/skills/confirmation_aw3/registration_db.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This package contains a the registration db model."""
+
+import datetime
+import json
+import logging
+import os
+import sqlite3
+from typing import Any, Dict, List, Tuple
+
+from aea.skills.base import Model
+
+
+_default_logger = logging.getLogger(
+    "aea.packages.fetchai.skills.confirmation_aw3.registration_db"
+)
+
+
+class RegistrationDB(Model):
```

Comment:
Please review carefully @marcofavorito

Reply:
differences from previous skill to look for?